controller-server/src/main/java/com/yahoo/vespa/hosted/controller/athenz/impl/SiaAthenzSslContextProvider.java

```diff
@@ -0,0 +1,22 @@
+package com.yahoo.vespa.hosted.controller.athenz.impl;
+
+import com.google.inject.Inject;
+import com.yahoo.container.jdisc.athenz.AthenzIdentityProvider;
+import com.yahoo.vespa.hosted.controller.api.integration.athenz.AthenzSslContextProvider;
+
+import javax.net.ssl.SSLContext;
+
+public class SiaAthenzSslContextProvider implements AthenzSslContextProvider {
```

Comment:
We should deprecate the `AthenzSslContextProvider` to signal that `AthenzIdentityProvider` should be used instead.